src/components/WebPlayBack.tsx

```diff
@@ -64,20 +66,56 @@ function WebPlayback(props: any) {
                 }
                 updatePlayerInfo(updatedPlayerInfo);
             });
-            
             player.connect();
-
             setPlayer(player);
         };
     }, []);
 
+    //method for updating the player objects.
     const updatePlayerInfo = (playerInfo: IPlayerUIProps) => {
         setPlayerInfo(playerInfo);
     }
+    //Method for playing and pausing music based on player state.
+    const toggle = (event: React.MouseEvent<HTMLElement>) => {
+        console.log("toggle clicked")
+        //@ts-ignore
+        statePlayer.getCurrentState().then(state => {
+            if (!state) {
+                //@ts-ignore
+                statePlayer.togglePlay().then(() => {
+                    console.log('player toggled: ' + state);
+                });
+            
+                return;
+            }
+            //@ts-ignore
+            statePlayer.togglePlay().then(() => {
+                console.log('player toggled: ' + state);
+            });
+        });
+    }
+    //Method for playing next track
+    const forward = (event: React.MouseEvent<HTMLElement>) => {
+        //@ts-ignore
+        statePlayer.nextTrack().then(() => {
+            console.log('Skipped to next track!');
```

Comment:
I like the use of the callback to log information! Once we start deploying this app we'll have to rethink how we're logging but this is good for now!

One thing to consider: we could also add a .catch in case the promise rejects and log the error it rejects with

Reply:
I didn't fix this. I like the way you are thinking I hadn't thought of that Thanks!